### PR TITLE
feat(osmosis-sync): added docker file for sync OSM with osmosis

### PR DESCRIPTION
**Pull request by galta95**

### CODE REVIEW

File: _sidebar.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ Dark Mode:
     * [Instructions](/osmosis/README.md)
     * [Resources](https://github.com/MapColonies/osm-depolyment-stack/tree/master/osmosis)
       * [Dockerfile](https://github.com/MapColonies/osm-depolyment-stack/blob/master/osmosis/Dockerfile)
+  * [osmosis-pipe-sync](/osmosis-pipe-sync/README.md)
+    * [Instructions](/osmosis-pipe-sync/README.md)
+    * [Resources](https://github.com/MapColonies/osm-depolyment-stack/tree/master/osmosis-pipe-sync)
```
Comment (syncush) [inline _sidebar.md:22]: should have its own section `Synchronization` -> `1 way` -> `Pipe based`

File: osmosis-pipe-sync/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  && cd /opt/osmosis \
+  && curl -L -o osmosis-$OSMOSIS_VERSION.tgz "https://github.com/openstreetmap/osmosis/releases/download/$OSMOSIS_VERSION/osmosis-$OSMOSIS_VERSION.tgz" \
+  && tar -xzvf osmosis-$OSMOSIS_VERSION.tgz \
+  && rm -f osmosis-$OSMOSIS_VERSION.tgz \
+  && ln -s /opt/osmosis/bin/osmosis /usr/local/bin/osmosis
+
+USER osmosis
+
+COPY ./start.sh .
+CMD ./start.sh
```
Comment (syncush) [inline osmosis-pipe-sync/Dockerfile:17]: You should use the `osmosis image` from the `osmosis` folder and not duplicate the `Dockerfile` content of `osmosis`

File: _sidebar.md (hunk 1 above)
Comment (galta95) [reply _sidebar.md:22]: Done

File: osmosis-pipe-sync/Dockerfile (hunk 2 above)
Comment (galta95) [reply osmosis-pipe-sync/Dockerfile:17]: Done

**Pull request merged**